paper/paper.md

```diff
@@ -53,14 +53,28 @@ Scientific data across experimental physics and materials science remains largel
 (Second sentence: We base everything on NeXus. We don't wanna say it's too complicated. Make it nicer. We make it more easier to handle it.)
 (Third: refer to cnxvalidate and such)
 
-# Dataconverter and validation (Sherjeel)
+## Dataconverter and validation (Sherjeel)
 
       * Mechanism to write an own reader, i.e. pynxtools-plugin mechanism and test frameworks, not mentioned to much about pynxtools-plugins
       * ELN Generator - one sentence or so
 
-# NeXus reader and annotator (read\_nexus) (Sandor)
+## NeXus reader and annotator (read\_nexus) (Sandor)
 
-# NOMAD integration (schema, parser) (Lukas, Sandor)
+## NOMAD integration (schema, parser) (Lukas, Sandor)
+
+While pynxtools works as a standalone tool using the command line, it can also be integrated directly into Research Data Management Systems (RDMS). Out of the box, the package functions as a plugin within the NOMAD platform[@Scheidgen:2023], converting and parsing data from experiments. This enables experimental data in the NeXus format to be integrated into NOMAD’s metadata model, making it searchable and interoperable with other data from theory and experiment. The plugin consists of several key components (called entry points):
+
+- Schema Package: The NeXus (meta)data definitions are natively defined in the NeXus Definition Language (NXDL) and expressed in XML. pynxtools converts this representation and extends NOMAD’s internal data schema (called __Metainfo__) with these domain-specific quantities. pynxtools also connects the NeXus vocabulary to existing base sections in NOMAD, which allows interoperability of NeXus-defined concepts with other standardized representations (e.g., for sample synthesis or theory) in NOMAD.
+
+- Data Converter: The __DataConverter__ as described above is also available in NOMAD. Hence, NOMAD users can directly convert their experimental data to NeXus using NOMAD's graphical interface. Aside from the capabilities already described, the internal __DataConverter__ class also handles NOMAD's electronic lab notebooks (ELNs) and converts these such that the manually inputted data can be converted to NeXus as well.
+
+- Parser: The NOMAD parser module in pynxtools (__NexusParser__) reads NeXus HDF5 files and uses the structured data instances from these files to populate the NOMAD __Metainfo__ model with __Metainfo__ object instances as defined by the pynxtools schema package. This step enables ingestion of NeXus data directly into the NOMAD __Metainfo__ model.
+
+- Normalization: Parsed data is post-processed using NOMAD’s normalization pipeline. This includes automatic handling of units (via pint), linking references across sections (including sample and instrument identifiers defined elsewhere in NOMAD), and populating derived quantities needed for advanced search and visualization.
```

Comment:
what makes the search "advanced", in my opinion the fact that it is not a mere text-based search but a semantic search in the sense that instances of NeXus concepts are searchable for, this makes it advances for sure.